proto/spec/api_collection.proto

```diff
@@ -48,6 +48,7 @@ message Parsing {
   string function_template = 2;
   BlockParser result_parsing = 3 [(gogoproto.nullable) = false];
   string api_name = 4;
+  string wanted_result = 5;

```

Comment:
change wanted_result to expected_result 

2. fix merge conflict with function_tag, as it was changed to Enum in omer's branch

Reply:
Fixed 4dc611b08e0ecf359f4bf505ce65013511829283